test/TestProjects/ProtocolMethodsInRestClient/Generated/TestServiceRestClient.cs

```diff
@@ -296,74 +365,6 @@ public virtual Response Delete(string resourceId, RequestContext context = null)
             }
         }
 
-        internal HttpMessage CreateGetRequest(string resourceId)
-        {
-            var message = _pipeline.CreateMessage();
-            var request = message.Request;
-            request.Method = RequestMethod.Get;
-            var uri = new RawRequestUriBuilder();
-            uri.Reset(_endpoint);
-            uri.AppendPath("/template/resources/", false);
-            uri.AppendPath(resourceId, true);
-            request.Uri = uri;
-            request.Headers.Add("Accept", "application/json");
-            return message;
-        }
-
-        /// <summary> Retrieves information about the resource. </summary>
-        /// <param name="resourceId"> The id of the resource. </param>
-        /// <param name="cancellationToken"> The cancellation token to use. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="resourceId"/> is null. </exception>
-        public async Task<Response<Resource>> GetAsync(string resourceId, CancellationToken cancellationToken = default)
-        {
-            if (resourceId == null)
-            {
-                throw new ArgumentNullException(nameof(resourceId));
```

Comment:
Why did we move all the protocol methods at the end? I think it is more convenient if we have this structure:
* Create request method for HLC
* Async HLC method
* Sync HLC method
* Create request method for protocol method
* Async version of  protocol method
* Sync version of protocol method

Instead of first all the HLC methods and then all the protocol methods.

Reply:
I don't know if we have any preferences. This file is codegened, it won't be edited by hand. Intellisence sorts everything alphabetically. From the before/after comparison perspective, it is easier to check added methods if they are in one place rather than spread across the file. 

@annelo-msft , what do you think?

Reply:
Fixed